pkgs/development/julia-modules/README.md

```diff
@@ -0,0 +1,2 @@
+The definitions in `default.nix` are generated with `julia2nix`. Use
```

Comment:
We need to be a bit more explicit here because julia2nix is already an occupied name https://github.com/codedownio/julia2nix

Reply:
I'll add a link to the file.